README.md

```diff
@@ -133,32 +133,33 @@ Supported languages
 
 Humanize Duration supports the following languages:
 
-| Language             | Code    |
-|----------------------|---------|
-| Arabic               | `ar`    |
-| Catalan              | `ca`    |
-| Chinese, simplified  | `zh_CN` |
-| Chinese, traditional | `zh_TW` |
-| Czech                | `cs`    |
-| Danish               | `da`    |
-| Dutch                | `nl`    |
-| English              | `en`    |
-| Finnish              | `fi`    |
-| French               | `fr`    |
-| German               | `de`    |
-| Greek                | `gr`    |
-| Hungarian            | `hu`    |
-| Italian              | `it`    |
-| Japanese             | `ja`    |
-| Korean               | `ko`    |
-| Norwegian            | `no`    |
-| Polish               | `pl`    |
-| Portuguese           | `pt`    |
-| Russian              | `ru`    |
-| Spanish              | `es`    |
-| Swedish              | `sv`    |
-| Turkish              | `tr`    |
-| Ukrainian            | `uk`    |
+| Language             | Code     |
+|----------------------|----------|
+| Arabic               | `ar`     |
+| Catalan              | `ca`     |
+| Chinese, simplified  | `zh_CN`  |
+| Chinese, traditional | `zh_TW`  |
+| Czech                | `cs`     |
+| Danish               | `da`     |
+| Dutch                | `nl`     |
+| English              | `en`     |
+| Finnish              | `fi`     |
+| French               | `fr`     |
+| German               | `de`     |
+| Greek                | `gr`     |
+| Hungarian            | `hu`     |
+| Italian              | `it`     |
+| Japanese             | `ja`     |
+| Korean               | `ko`     |
+| Lithuanian           | `lt`     |
+| Norwegian            | `no`     |
+| Polish               | `pl`     |
+| Portuguese           | `pt`     |
+| Russian              | `ru`     |
+| Spanish              | `es`     |
+| Swedish              | `sv`     |
+| Turkish              | `tr`     |
+| Ukrainian            | `uk`     |
```

Comment:
Damn... There is aditional space at the end. I was added a short version of "lt" language - "shortLt". And the table was to short. But later I decided to remove it (the "shortLt") and forgot to shrink the table.